packages/base-driver/lib/helpers/capabilities.js

```diff
@@ -20,6 +20,50 @@ function isW3cCaps (caps) {
   return false;
 }
 
+function fixCaps (originalCaps, desiredCapConstraints = {}, log) {
```

Comment:
no type definitions in this function? is there a reason we're not ts-checking this file?